catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/AbstractEventController.java

```diff
@@ -230,7 +230,9 @@ public void registerUserSession(final ServerSession serverSession, ServerMessage
 
         userSessionMap.put(userId, serverSession);
 
-        LOGGER.debug("Added ServerSession to userSessionMap - New map: {}", userSessionMap);
+        LOGGER.debug("Added ServerSession for user {} to userSessionMap - New map: {}",
```

Comment:
@figliold should be fine. @oconnormi  made the same formatting changes in https://github.com/codice/ddf/pull/1027